silver/api/urls.py

```diff
@@ -41,6 +41,8 @@
         views.PaymentMethodList.as_view(), name='payment-method-list'),
     url(r'^customers/(?P<customer_pk>[0-9]+)/payment_methods/(?P<payment_method_id>[0-9]+)/$',
         views.PaymentMethodDetail.as_view(), name='payment-method-detail'),
+    url(r'^customers/(?P<customer_pk>[0-9]+)/payment_methods/(?P<payment_method_id>[0-9]+)/(?P<requested_action>[\w]+)_request/$',
```

Comment:
I suggest we either hardcode every possible action (maybe with a regex) and / or validate `requested_action` in `PaymentMethodAction`.